### ISSUE TITLE
The GitHub build instructions in testflight.md needs updating for Trio app identifier names etc

### ISSUE THREAD

**Issue by bjornoleh** [nightscout/Trio#294]
## Describe the bug
The current GitHub build instructions in testflight.md needs updating for Trio app identifier names etc to reflect the actual names and what it currently looks like in the Apple Developer portal.

There is a mismatch between the following section and the screenshots below:
https://github.com/bjornoleh/Trio/blob/trio-bo/fastlane/testflight.md#add-app-group-to-bundle-identifiers

> ## Add App Group to Bundle Identifiers
>
> 1. Go to [Certificates, Identifiers & Profiles](https://developer.apple.com/account/resources/identifiers/list) on the Apple developer site.
> 1. For each of the following identifier names:
    * FreeAPS
    * FreeAPS watchkitapp
    * FreeAPS watchkitapp watchkitextension

![image](https://github.com/nightscout/Trio/assets/63544115/1f4c1ff8-cdb0-47dd-95a3-99cef9f724cb)

![image](https://github.com/nightscout/Trio/assets/63544115/160ef9bc-fc39-490e-adc9-29c5e24d1a96)

## Additional context
I had previously built the app with Xcode, my screenshots probably differ from a GitHub actions only builder, as I believe the `XC` prefix indicates Xcode generated identifiers.

Hopefully someone can supplement with relevant screnshots.

### Trio Version:
* Repo: *nightscout/trio*
* Git Reference: [32b5e95](https://github.com/nightscout/Trio/commit/32b5e95377e085b5c641552b96023d70354e6a26)

**Comment by bjornoleh** [comment]
Tagging @marionbarker for her expertise in this area.

Also noting that the instructions in https://docs.diy-trio.org/en/latest/operate/build.html#build-trio-with-github appear to be accurate:

```
Identifier Names will be: FreeAPS, FreeAPSWatch, FreeAPSWatch WatchKit Extension, LiveActivityExtension

These names are the same as that used by iAPS, they are distinguised by the Identifier itself, which include the new BundleID for your Trio app

If you build with Xcode before, they will start with XC and use the Identifier string rather than the Name

Identifiers will be:

org.nightscout.TEAMID.trio

org.nightscout.TEAMID.trio.watchkitapp

org.nightscout.TEAMID.trio.watchkitapp.watchkitextension

org.nightscout.TEAMID.trio.LiveActivity

Only the first 3 of these identifiers need to have the App Group added

For the list below, make sure the box next to App Group is checked, and click Configure to select the same app group as Loop: group.com.TEAMID.loopkit.LoopGroup

You must add this App Group to each of the following identifiers:

org.nightscout.TEAMID.trio

org.nightscout.TEAMID.trio.watchkitapp

org.nightscout.TEAMID.trio.watchkitapp.watchkitextension

In App Store Connect, the Bundle ID for Trio will be: org.nightscout.TEAMID.trio
```

**Comment by marionbarker** [comment]
I closed this because #297 was merged.
